source/controller/src/map_ctrl_main.c

```diff
@@ -142,7 +146,7 @@ int main(int argc, char *argv[])
         sig_stop_action.sa_flags = 0;
         sigaction(SIGTERM, &sig_stop_action, NULL);
         sigaction(SIGINT, &sig_stop_action, NULL);
-        sigaction(SIGSEGV, &sig_stop_action, NULL);
+        // sigaction(SIGSEGV, &sig_stop_action, NULL);
```

Comment:
This is to enable breakpad minidumps

Reply:
Please add Breakpad support, after rdk-easymesh-controller.bb recipe file is merged.